chirp/drivers/kenwood_d7.py

```diff
@@ -0,0 +1,344 @@
+# Copyright 2010 Dan Smith <dsmith@danplanet.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from chirp.drivers.kenwood_live import KenwoodOldLiveRadio, MODES, STEPS, \
+    DUPLEX, LOG, get_tmode
```

Comment:
Importing LOG is technically not correct because that logger will log as `kenwood_live` and not `kenwood_d7` but it's not critical. Just fix that up in your settings series.

Reply:
Heh, holding off on another force push with the proper fix. :)

Reply:
I already merged this as you had it. It's fine, just fix it up in your next one. I wanted to make sure we merged this split before I leave because it'll be easier for you to iterate on the settings stuff with it done. Thanks for the swift action :)